lib/builtins/int_lib.h

```diff
@@ -89,7 +89,7 @@
  * Presumably it's any version of GCC, and targeting an arch that
  * does not have dedicated bit counting instructions.
  */
-#if (defined(__sparc64__) || defined(__sparcv9) || defined(__mips_n64) || defined(__mips_o64) || defined(__riscv__) \
+#if ((defined(__sparc__) || defined(__arch64__)) || defined(__mips_n64) || defined(__mips_o64) || defined(__riscv__) \
```

Comment:
Shouldn’t that be `&&`? Though I wonder whether 32-bit SPARC would also need it and thus if the `defined(__arch64__)` can be dropped...

Reply:
Yes, it should -.-.